docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -415,77 +429,100 @@ Finally
 GPU Policies for SYCL
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
- ======================================== ============= ========================
+ ======================================== ============= ==============================
  SYCL Execution Policies                  Works with    Brief description
- ======================================== ============= ========================
+ ======================================== ============= ==============================
  sycl_exec<WORK_GROUP_SIZE>               forall,       Execute loop iterations
                                                         in a GPU kernel launched
                                                         with given work group
                                                         size.
+ sycl_launch_t                            launch        Launches a sycl kernel,
+                                                        any code express within
+							the lambda is executed
+							on the device.
  sycl_global_0<WORK_GROUP_SIZE>           kernel (For)  Map loop iterates
                                                         directly to GPU global
                                                         ids in first
-                                                        dimension, one iterate 
+                                                        dimension, one iterate
                                                         per work item. Group
                                                         execution into work
-                                                        groups of given size. 
+                                                        groups of given size.
  sycl_global_1<WORK_GROUP_SIZE>           kernel (For)  Same as above, but map
                                                         to global ids in second
                                                         dim
  sycl_global_2<WORK_GROUP_SIZE>           kernel (For)  Same as above, but map
-                                                        to global ids in third 
+                                                        to global ids in third
                                                         dim
+ sycl_global_item_0                       launch (loop) Creates a unique thread
```

Comment:
What's different between sycl_global_0 and sycl_global_item_0?

Reply:
ah! I didn't know we had that. At the moment sycl_global_0 takes an optional template parameter for the work_group_size -- along the same idea of what you are doing here: https://github.com/LLNL/RAJA/pull/1334 . But for now it does not work with launch, I think this is something to address along with your current PR.